DOCUMENTATION.md

```diff
@@ -0,0 +1,150 @@
+Base Configuration
+</h2>
+
+<br>
+
+### What's included?
+The base configuration covers most popular languages (Python, JS/Node.js, Ruby, Go are all included), sorry Max - no Rust yet! But each configuration setup included looks for best practise dependency tracking files in each project (`package.json` for Node.js, `requirements.txt` for Python projects). This can also be customized locally if your team is using nonconventional names for your tracking files.
+
+<br>
+
+Each repo will also have their own Dependency Dashboard gently lurking in their Issues board on Github. This dashboard shows which updates are waiting to be addressed by developers in the workflow agreed upon by each team (Scrum, Kanban, Sprint etc). When develpers create branches and PRs from the Dependency Dashboard they can then see the most up to date information available to help them integrate the updates. 
+
+<br>
+
+This Dashboard is updated on a weekly basis so that teams don't feel overwhelmed by a constant and neverending list of dependency PRs!
+
+<br>
+
+We only allow a maximum of 10 PRs to be open on the repo at any one time. This minimizes the feeling of work never quite completing since open PRs that can't be addressed in a timely manner end up feeling like meaningless work
+
+<br>
+
+Groups some like dependencies together, such as linters. You can also customize other groupings that make sense for you and your team. This also minimizes context switching.
+
+<br>
+
+Automatically rebases stale PRs, which are defined as those lasting longer than 21 days. This should be enough time to address major or critical updates. If an update is deemed infeasable for the time being you can move it into a temporary `ignore` state. However we don't recommend this. This option should be for patches that can't be fixed until the upstream maintainers fix a critical bug or issue.
+
+<br>
+
+Uses semantic PR prefixes (for example `fix-`, `bug-`, or `chore-`)
+
+<br>
+
+Updates Docker images on major updates only as well as pins image digests instead of tags. Tags are just pointers that can be maliciously updated to point to bad images, whereas digest SHA IDs ensure you're consistenly pulling down secure and reliable images.
+
+<br>
+
+Automatically pins the Renovate Github Action to it's respective digest as well, ensuring we're always using the offical Renovate Bot action!
+
+<br>
+
+PR creation is set to immediate meaning as soon as a branch with the updated dependencie is created so is the PR. It is recommended you do not override this value. You can set this to either create a PR upon successful repo status checks or passiing tests but this can be brittle and undermine the point of automatic dependency management. There is also a manual setting to require human approval, but this will create a bottleneck in your PR process. Note this means approval to create a PR, not like a code approval to merge an already existing one! Adding an additional step that may not be useful.
```

Comment:
Also curious about this one since I thought it was that PRs would only be created when:

- The branch is created based on our `schedule` config
- The stability days requirement is met

Reply:
Oops, and it's set to `not-pending` not `immediate` meaning all status checks must complete before the PR is raised. Updated the text.